rskj-core/src/main/java/org/ethereum/rpc/exception/RskErrorResolver.java

```diff
@@ -23,10 +25,21 @@ public JsonError resolveError(Throwable t, Method method, List<JsonNode> argumen
             error =  new JsonError(((RskJsonRpcRequestException) t).getCode(), t.getMessage(), null);
         } else if (t instanceof InvalidFormatException) {
             error = new JsonError(-32603, "Internal server error, probably due to invalid parameter type", null);
+        } else if (t instanceof UnrecognizedPropertyException) {
+            String message = getExceptionMessage((UnrecognizedPropertyException) t);
+            logger.error("JsonRPC error {} for method {} with arguments {}", message, method, arguments);
+            error = new JsonError(-32603, message, null);
```

Comment:
Maybe we could use a constant for this value? Since it's used several times in this file (and others)

Reply:
I like it! 🙌🏼

Reply:
Done!